Makefile

```diff
@@ -56,11 +56,20 @@ ifeq (${REGION},)
 $(error region must be set in command line using REGION variable or config files)
 endif
 
-# URL_SUFFIX - used for the docker login
+ifeq (${PARTITION},)
```

Comment:
Is there not a way to get the partition or domain from CDK like we do in `core/utils.ts`

```
.replace(/\${AWS::Partition}/gi, cdk.Aws.PARTITION)
```

Reply:
This is clearly using the ts library but wasn't sure if we could use the command line tool to figure this out since we already require customers to supply the region

Reply:
I can retrieve the partition using AWS cli within the makefile, but it does add that as a new dependency (which presumably is already installed).